package.json

```diff
@@ -20,7 +20,11 @@
     "storybook": "start-storybook -p 4444"
   },
   "lint-staged": {
-    "*.js": ["eslint --fix", "prettier --write", "git add"]
+    "*.js": [
```

Comment:
we need to avoid prettifying package.json in our editors, or we are gonna be stuck in an infinite commited lines for nothing